packages/rs-drive-proof-verifier/src/error.rs

```diff
@@ -2,6 +2,7 @@ use dpp::ProtocolError;
 
 /// Errors
 #[derive(Debug, thiserror::Error)]
+#[cfg_attr(feature = "mocks", derive(bincode::Encode, bincode::Decode))]
```

Comment:
We use serde in dapi client

Reply:
It's all under `mocks` feature atm, we should introduce separate `serde` feature. TBD in next PR.